hibernate-core/src/main/java/org/hibernate/dialect/H2Dialect.java

```diff
@@ -138,16 +133,10 @@ public H2Dialect(DatabaseVersion version) {
 		// 1.4.200 introduced changes in current_time and current_timestamp
 		useLocalTime = version.isSameOrAfter( 1, 4, 200 );
 
-		if ( version.isSameOrAfter( 1, 4, 32 ) ) {
-			this.sequenceInformationExtractor = version.isSameOrAfter( 1, 4, 201 )
-					? SequenceInformationExtractorLegacyImpl.INSTANCE
-					: SequenceInformationExtractorH2DatabaseImpl.INSTANCE;
-			this.querySequenceString = "select * from INFORMATION_SCHEMA.SEQUENCES";
-		}
-		else {
-			this.sequenceInformationExtractor = SequenceInformationExtractorNoOpImpl.INSTANCE;
-			this.querySequenceString = null;
-		}
+		this.sequenceInformationExtractor = version.isSameOrAfter( 1, 4, 201 )
+				? SequenceInformationExtractorLegacyImpl.INSTANCE
+				: SequenceInformationExtractorH2DatabaseImpl.INSTANCE;
```

Comment:
Please remove the field and simply return `SequenceInformationExtractorLegacyImpl.INSTANCE` from the `getSequenceInformationExtractor` method.

Reply:
Are you sure about this change? the minimum version will be 1.4.197, i.e. could be less than the 201 that `SequenceInformationExtractorLegacyImpl`requires.

Reply:
Oops, yeah you are right. Looked at the wrong version.